app/presenters/reservation_calendar.rb

```diff
@@ -0,0 +1,50 @@
+class ReservationCalendar < SimpleDelegator
+
+  attr_accessor :url, :ical
+
+  def self.to_calendar(reservation, url = nil)
+    new(reservation, url).to_ical
+  end
+
+  def initialize(reservation, url = nil)
+    super(reservation)
+    @url = url
+    generate_ical
+  end
+
+  def generate_ical
+    @ical = Icalendar::Calendar.new
+    ical.event do |e|
+      e.dtstart = Icalendar::Values::DateTime.new(reserve_start_at)
+      e.dtend = Icalendar::Values::DateTime.new(reserve_end_at)
+      e.summary = summary
+      e.description = description
+      e.location = facility.name
+      e.url = url if url.present?
```

Comment:
Will the controller be responsible for wrapping the URL in whatever wrapper Icalendar needs? (we can deal with that when we need to, I suppose)